src/Commands/core/SecurityUpdateCommands.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace Drush\Commands\core;
+
+use Composer\Semver\Comparator;
+use Drush\Commands\DrushCommands;
+use Drush\Drush;
+use GuzzleHttp\Client;
```

Comment:
I think this would be a new dependency. I'm not too excited about it when we have the shell at our disposal. Maybe borrow the bits from http://api.drush.org/api/drush/includes%21drush.inc/function/_drush_download_file/8.0.x. Thats no longer in Drush core so you could just keep it local to this command for now.

Reply:
For such a simple use-case as this, `file_get_contents($url);` would suffice.

Reply:
Yeah my bad, I thought that this was an existing dependency because PHPStorm was registering it, but I see now that it was not a drush dependency but rather a dependency in my test project.